docs/reviewing.md

```diff
@@ -0,0 +1,36 @@
+# Pull request and issue management
+
+## Merging pull requests
+
+> [!NOTE]
+> This information is for [project peers and owners](../GOVERNANCE.md#roles-and-responsibilities).
+
+If a pull request has been approved and tests are passing, then you can merge it.
+Follow these steps to merge:
+
+1. Make sure you **should** merge this pull request.
+   In general, only merge pull requests you are already familiar with, either as an author or reviewer.
+
+1. Make sure **you** should merge this pull request.
+   Some changes—most notably, feature removals, tooling and policy changes, and schema changes—require an owner to merge.
+   See [the privileges and responsibilities matrix](../GOVERNANCE.md#privileges-and-responsibilities-matrix) for more information.
+
+1. Make sure there are no explicit blockers to merging.
+   Blockers include:
+
+   - The pull request has the [blocked label](https://github.com/web-platform-dx/web-features/pulls?q=is%3Aopen+is%3Apr+label%3Ablocked)
+   - A reviewer asked to re-review before merging or has requested changes that have not been addressed with a commit or comment
+   - The author explicitly asked to delay merging
+   - The author explicitly asked for review from all requested reviewers (the presumption is at least one but not all)
+
+   If a blocker no longer applies, remove the label or dismiss the stale review.
```

Comment:
Can you explain what you mean by "dismiss the stale review"? And how would someone do this in github?

Reply:
Down in the merge UI, there's a widget that shows approving or changes requested reviews. But apparently this is not well-known, so I'll link to the GitHub docs.

```suggestion
   If a blocker no longer applies, remove the label or [dismiss the stale review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/dismissing-a-pull-request-review).
```